drivers/tps6274x/tps6274x.c

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2017 RWTH Aachen
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+
+#include "tps6274x.h"
+#define ENABLE_DEBUG        (0)
+#include "debug.h"
+
+static void print_err(char *, uint8_t *, uint8_t, uint8_t *);
```

Comment:
can this line be removed? need to add "static" below where print_err is defined.